avocado/core/result.py

```diff
@@ -96,7 +100,7 @@ def __init__(self, job):
         """
         self.job_unique_id = getattr(job, "unique_id", None)
         self.logfile = getattr(job, "logfile", None)
-        self.tests_total = getattr(job.args, 'test_result_total', 1)
+        self.tests_total = 0
```

Comment:
Btw how about using `avocado run --dry-run --json -` to verify the execution and get the number of tests instead of `avocado list`?